src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml.cs

```diff
@@ -117,22 +122,37 @@ private void OnTextAreaTextEntered(object sender, TextCompositionEventArgs e)
 
         private void OnAcceptClicked(object sender, RoutedEventArgs e)
         {
-            DialogResult = true;
+            UpdateScript(editText.Text);
+            this.Close();
         }
 
         private void OnCancelClicked(object sender, RoutedEventArgs e)
```

Comment:
Can we rename this to `OnRevertClicked` just so that the new intent is clearer or was `Cancel` doing the same thing before?